unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts

```diff
@@ -166,7 +166,7 @@ export function DynamicFacetValueTest() {
       });
 
       it('should return the correct aria-label', () => {
```

Comment:
I would remove this test, since it's repetitive with the other one now

Reply:
good point. Removed it.